spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb

```diff
@@ -143,7 +143,7 @@
 
           expect do
             RapidReadyForDecision::DisabilityCompensationJob.perform_async(submission_without_account_or_edpid.id)
-          end.to raise_error RapidReadyForDecision::DisabilityCompensationJob::AccountNotFoundError
+          end.to raise_error Form526RapidReadyForDecisionConcern::AccountNotFoundError
```

Comment:
Note that `DisabilityCompensationJob` is only used as a backup Sidekiq job: https://github.com/department-of-veterans-affairs/vets-api/blob/01e46b8bdf62e8bb940e93459446297cfa017082/app/services/rapid_ready_for_decision/constants.rb#L12

`DisabilityCompensationJob` will eventually be removed.
Let's not modify it or associated tests as that could be interpreted as: the class is being actively maintained.